_docs/installation.md

```diff
@@ -0,0 +1,203 @@
+
+**source:** https://github.com/cga-harvard/HHypermap/README.md
+
+## Installation
+
+### Manual Installation
+
+Using ubuntu 14.04
+
+```sh
+sudo apt-get update
+sudo apt-get install postgresql rabbitmq-server python-virtualenv git python-psycopg2 libjpeg-dev python-dev libxml2-dev libxslt-dev libxslt1-dev libpq-dev libgeos-dev
+```
+
+Create PostgreSQL database.
+
+```sh
+sudo -u postgres psql
+CREATE DATABASE hypermap;
+CREATE USER hypermap WITH superuser PASSWORD 'hypermap';
+\q
+```
+
+Install java8 for elasticsearch
+```sh
+sudo add-apt-repository ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get install oracle-java8-installer
+```
+
+Install and configure elasticsearch
+
+```sh
+wget -qO - https://packages.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+echo "deb https://packages.elastic.co/elasticsearch/2.x/debian stable main" | sudo tee -a /etc/apt/sources.list.d/elasticsearch-2.x.list
+sudo apt-get update && sudo apt-get install elasticsearch
+sudo sed -i -e 's/#ES_HEAP_SIZE=2g/ES_HEAP_SIZE=1g/' /etc/default/elasticsearch
+sudo service elasticsearch start
+```
+
+Install registry on a virtual environment.
+
+```sh
+virtualenv --no-site-packages env
+source env/bin/activate
+git clone https://github.com/cga-harvard/HHypermap.git
+cd HHypermap
+git checkout registry
+pip install -e .
+```
+Create environment variables. 
+
+```sh
+#!/bin/bash
+export DJANGO_SETTINGS_MODULE=hypermap.settings
+export REGISTRY_SEARCH_URL=elasticsearch+http://elasticsearch:9200/
+export DATABASE_URL=postgres://postgres:postgres@postgres:5432/postgres
+export BROKER_URL=amqp://guest:guest@rabbitmq:5672/
+export ALLOWED_HOSTS=['django',]
+export C_FORCE_ROOT=1
+export SEARCH_MAPPING_PRECISION=50m
+export CELERY_DEFAULT_EXCHANGE=hypermap
+```
+
+Execute migrations.
+
+```sh
+python manage.py migrate
+```
+
+Finally, load fixtures
+```sh
+python manage.py loaddata hypermap/aggregator/fixtures/catalog_default.json
+python manage.py loaddata hypermap/aggregator/fixtures/user.json
+```
+
+## Integration within a Django project
+
+Make sure you have registry module installed inside your working environment.
+```
+pip install django-registry
+```
+
+Add this variables into your settings.py
+
+```python
+from hypermap.settings import REGISTRY_PYCSW
+
+REGISTRY = True # ensure the value is True
+REGISTRYURL = '%s/registry' % SITE_URL.rstrip('/')
+CATALOGLIST = [
+    {
+        "name": "local registry",
+        "url": "%s/hypermap/" % SITE_URL.rstrip("/")
+    },
+]
+SEARCH_ENABLED = True
```

Comment:
We should read from os.getenv, as in master. I think this happens also for other settings